### PR TITLE
Added support for shareable pages. Added embedding connections doc.

### PR DESCRIPTION
**Pull request by n0mid**
Added documentation for Embedding Connections 

Documentation link is `oem/embedding-connections.html`.

Снимок экрана 2019-10-21 в 13 22 19

### CODE REVIEW

File: oem/embedding-connections.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    function receiveMessage(event) {
+       var data = JSON.parse(event.data);
+       
+       switch (data.type) {
+           case 'heightChange':
+              document.getElementById('workatoId').style.height = data.payload.height + 'px';
+              break;
+           case 'connectionStatusChange':
+               var message = data.payload.connected ? 'Connected' : 'Disconnected';
```
Comment (kandadaboggu) [inline oem/embedding-connections.md]: Fix the alignment of the code.

File: oem/embedding-connections.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+               break;
+   
+       }
+    }
+  </script>
+</head>
+
+<body>
+<h4>Status: <span id="statusId"></span></h4>
+<iframe id="workatoId" src="https://workato.com/direct_link/embedding_connections/<connection_id>>?workato_dl_token=<jwt_token>" style="width: 500px; height: 150px; border: 0"></iframe>
```
Comment (kandadaboggu) [inline oem/embedding-connections.md]: Can we examples of how dl_token can be generated?

File: oem/embedding-connections.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+Widget API works on postMessage. 
+
+Format: `{type: string, payload: object}` 
+
+Is supported next types: 
+
+| Type  | Payload | Description |
+| ------------- | ------------- | -------- |
+| heightChange  | { height: number } | Content height was changed
+| connectionStatusChange  | { connected: string, connectionId: number, provider: string } | Connection status was changed
```
Comment (kandadaboggu) [inline oem/embedding-connections.md]: - We should return the error message if any.
- What are the possible values for connected? 
- Can we make the payload similar to the payload we return in our public API's

```ruby
{
  id: 123,
  provider: "salesforce",  
  authorization_status: "failure", #success/failure
  authorization_error: "Unauthorized"
}
```
https://docs.workato.com/workato-api/managed-users.html#list-customer-connections

Comment (n0mid) [reply oem/embedding-connections.md]: My mistake, `connected` is a boolean. 

Something like this? 
```
{
  id: <connection_id>,
  provider: <provider>,  
  connection_status: <connected/disconnected>
  connection_error: <err_message/null>
}
```

Comment (kandadaboggu) [reply oem/embedding-connections.md]: okay. Looks fine

Comment (kandadaboggu) [reply oem/embedding-connections.md]: You can use `connected` (true/false), do add a field for `error`.

**Pull request merged**